### DISCUSSION TITLE
Content preferences not deleted

### DISCUSSION THREAD

**Discussion by notDavid** [Cookie-AutoDelete/Cookie-AutoDelete#1048] [Q&A / Support] [answered]
Hi,

it seems "Site-specific preferences" are not deleted. These are stored in `permissions.sqlite` and
`content-prefs.sqlite` and store things like for instance, which sites are allowed to display popups or zoom levels that are set on a site-by-site basis.

Is this correct?

Would it be useful to add this to Cookie Autodelete, so things like permissions for location, notifications, etc. can be cleared upon browser close for not-whitelisted websites?

Thanks.

**Comment by kennethtran93** [comment, marked as answer]
I remember seeing Chrome extensions having this API to a certain extent, but not on Firefox yet.

**Reply by notDavid** [reply]
Thanks for your reply, good to know... :)
